sencha/translate.py

```diff
@@ -29,6 +29,24 @@
 logger = get_logger(__file__)
 
 
+def calculate_chunksize(total_jobs_todo, processes):
+    """
+    Return a generator of strings after
+    splitting a string by the given separator
+
+    sep : str
+        Separator between strings, default one space
+    Returns
+    -------
+    Yields generator of strings after
+    splitting a string by the given separator
+    """
+    chunksize, extra = divmod(total_jobs_todo, processes)
+    if extra:
+        chunksize += 1
+    return chunksize
```

Comment:
should this by yield then instead of return if it yields a generator of strings?

Reply:
oops incorrect docstring - corrected now